src/mock-graphql/resolvers.ts

```diff
@@ -1,9 +1,10 @@
-import { getSubmissions, startSubmission } from '../use-cases';
+import { getSubmissions, startSubmission, getCurrentUser } from '../use-cases';
 
 const submissions = [];
 
 export const resolvers = {
     Query: {
+        getCurrentUser: getCurrentUser(),
```

Comment:
Let's call this `getCurrentProfile`

Reply:
done. should we also change the graphql type from User to Profile?